### ISSUE TITLE
Why training an ITER is over

### ISSUE THREAD

**Issue by XXMxxm220** [CoinCheung/BiSeNet#307]
Why training an ITER is over？

**Comment by XXMxxm220** [comment]
训练一个epoch后  不再继续

**Comment by zhanghaowei01** [comment]
您好，问题解决了没

**Comment by XXMxxm220** [comment]
> 您好，问题解决了没

解决啦

**Comment by zhanghaowei01** [comment]
分享一下呗

**Comment by XXMxxm220** [comment]
> 分享一下呗

你好，我采用的方法是在for it 前，加入了for epoch in range epochs：；epochs是我自己设定的训练周期，再将save_model语句也包含在for epoch 内，保证每个epch保存一次模型。

**Comment by CoinCheung** [comment]
Why do you need this? You can compute the total iterations by `len(dataset) * n_epoches / batch_size`, and set the total iterations as this computation result.

**Comment by XXMxxm220** [comment]
> Why do you need this? You can compute the total iterations by , and set the total iterations as this computation result.`len(dataset) * n_epoches / batch_size`

You are right .Thank you  for your response.I just wanted to use epoch to mean a training, not iters，so I choose that way. This approach also does not affect the final output
